src/nvim/lua/stdlib.c

```diff
@@ -606,6 +607,10 @@ void nlua_state_add_stdlib(lua_State *const lstate, bool is_thread)
     lua_pushcfunction(lstate, &nlua_iconv);
     lua_setfield(lstate, -2, "iconv");
 
+    // vim.base64
+    luaopen_base64(lstate);
+    lua_setfield(lstate, -2, "base64");
```

Comment:
We already have `vim.json` and `vim.pack`, so yeah, it's hard to say whether we should introduce e.g. `vim.codec` ([python module](https://docs.python.org/3/library/codecs.html) for reference).